horizons/gui/tabs/productiontabs.py

```diff
@@ -332,3 +332,31 @@ def get_displayed_productions(self):
 		all_farm_productions = self.instance.get_component(Producer).get_productions()
 		productions = {p for p in all_farm_productions for res in p.get_consumed_resources().keys() if res in possible_res}
 		return sorted(productions, key=operator.methodcaller('get_production_line_id'))
+
+class DoctorOverviewTab(ProductionOverviewTab):
+	"""Same but with extra number of effected buildings"""
+	widget = 'overview_doctor.xml'
+
+	def init_widget(self):
+		super().init_widget()
+		self.update_data()
+
+	def update_data(self):
+		field_comp = self.instance.get_component(FieldBuilder)
+		label = self.widget.child_finder('affected_buildings')
+
+		label.text = 'niggerFaggot'
+
+class FireStationOverviewTab(ProductionOverviewTab):
```

Comment:
Same as with the xml definitions: can these be abstracted in a single class?

Even if this would not be possible it's still better to make a superclass for doctor and firestation and then have the separate classes inherit from them and then only have the differences